src/bci_build/package/templates/access_protected_images.j2

```diff
@@ -0,0 +1,37 @@
+## Accessing the Container Image
+
+To access this image container image requires a valid SUSE subscription. In order
+to access the container image, you must login to the SUSE Registry with your credentials. There are three ways to do that.
```

Comment:
I don't want to do a numbered listing here, as it might be confusing others that they need to do all three as steps in order, while only one of them is sufficient. 

I've reworded it slightly

Reply:
Looks good.